template/main.typ

```diff
@@ -53,12 +53,12 @@ These functions will overwrite the standard #link("https://typst.app/docs/refere
 Here is how Typst's own #std-smallcaps[smallcaps] and #std-upper[upper] look compared to the 'Ilm's variants:\
 #hide[Here is how Typst's own ] #smallcaps[smallcaps] and #upper[upper]
 
-They both look similar, the only difference is that 'Ilm uses more spacing between individual characters.
+They both look similar, the only difference being that 'Ilm uses more spacing between individual characters.
 
-If you prefer Typst's default spacing then you can still use it by prefixing `std-` to the functions: ```typst #std-smallcaps()``` and ```typst #std-upper()```.
+If you prefer Typst's default spacing, you can still use it by prefixing `std-` to the functions: ```typst #std-smallcaps()``` and ```typst #std-upper()```.
 
 == Footer
-If a page does not begin with a chapter then we display the chapter's name, to which the current section belongs to, in the footer. #link(<demo>)[Click here] to go to @demo down below and see the footer in action.
+If a page does not begin with a chapter, then we display the chapter's name, to which the current section belongs, in the footer. #link(<demo>)[Click here] to go to @demo down below and see the footer in action.
```

Comment:
Here I am using the comma with a relative clause. See the section for relative clause on https://www.grammarly.com/blog/comma/ for reference.

Kindly revert this change.